instagram.py

```diff
@@ -36,17 +41,15 @@ def login_instagram(username, password):
         cookies = login_response.cookies
         cookie_jar = cookies.get_dict()
         csrf_token = cookie_jar['csrftoken']
-        print("csrf_token: ", csrf_token)
         session_id = cookie_jar['sessionid']
-        print("session_id: ", session_id)
+        return csrf_token, session_id
     else:
         print("login failed ", login_response.text)
 
 
-def share_photo(photo):
+def share_photo(photo, caption):
     micro_time = int(datetime.now().timestamp())
-    csrf_token = "Place the csrf_token here"
-    session_id = "place the session_id here"
+    csrf_token, session_id = login_instagram(username, password)
```

Comment:
Thank you for your commit. but you need to consider that once you login the csrf_token and session_id is valid for a couple of months so you don't need to log in each time you want to share a post.